plugins/basic/python/tk_maya_basic/plugin_engine.py

```diff
@@ -143,20 +109,15 @@ def shutdown():
 
     # Re-import the toolkit core to ensure usage of a swapped in version.
     import sgtk
-
     logger = sgtk.LogManager.get_logger(PLUGIN_PACKAGE_NAME)
-
     engine = sgtk.platform.current_engine()
 
     if engine:
-
-        logger.info("Stopping the %s engine." % manifest.engine_name)
-
+        logger.info("Stopping the Shotgun engine.")
         # Close the various windows (dialogs, panels, etc.) opened by the engine.
         engine.close_windows()
-
         # Turn off your engine! Step away from the car!
         engine.destroy()
 
     else:
-        logger.warning("The %s engine is already stopped!" % manifest.engine_name)
+        logger.debug("The maya engine was already stopped!")
```

Comment:
Should this read "The Shotgun engine was already stopped!" to be consistent with line 116 (or vice versa)?